recipes/aimodelshare/meta.yaml

```
@@ -0,0 +1,64 @@
+{% set name = "aimodelshare" %}
+{% set version = "0.0.94" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 36b35968e0605fa57610eee39b568a491666ed4d76b5287ea8d276fb83294dd4
+
+build:
+  skip: True  # [unix]
```

The selectors make this line only appear if the expression evaluates to true so you want:
```suggestion
  skip: True  # [win]
```